projects/canopy/src/lib/forms/radio/radio-button.component.spec.ts

```diff
@@ -1,17 +1,60 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
+import { Component, DebugElement } from '@angular/core';
+import {
+  FormBuilder,
+  FormGroup,
+  FormsModule,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
 
 import { anything, instance, mock, verify, when } from 'ts-mockito';
+import { MockComponents } from 'ng-mocks';
 
 import { LgErrorStateMatcher } from '../validation/error-state-matcher';
 import { LgRadioButtonComponent } from './radio-button.component';
 import { LgRadioGroupComponent } from './radio-group.component';
+import { LgHintComponent } from '../hint';
+
+const hintTestId = 'test-hint-id';
+
+@Component({
+  template: `
+    <form [formGroup]="form" #testForm="ngForm">
+      <lg-radio-group formControlName="color">
+        Color
+        <lg-radio-button value="red">Red</lg-radio-button>
+        <lg-radio-button value="yellow"
+          >Yellow
+          <div class="lg-radio-button__content">
+            <lg-hint id="${hintTestId}">Custom text</lg-hint>
```

Comment:
Do you need to add an ID here? I think the hint component adds one automatically.

Reply:
Yes, it seems to be needed for the last assertion where we check if the id matches with the value of the `aria-describedBy` attribute, however; I've had some inspiration on the tests from this file (https://github.com/Legal-and-General/canopy/blob/master/projects/canopy/src/lib/forms/radio/radio-group.component.spec.ts#L35) where I've seen that the approach is similar 😉 . 

Indeed when the component is being generated in a project, that id will be automatically created 👍 .

Reply:
Ah, gotcha, thanks 👍